README.md

```diff
@@ -1,59 +1,124 @@
+
+### Running the live site locally (Bazel)
+
+```console
+$ bazel run //site:live
+```
+
+The site should now be available by visiting http://localhost:4000.
+
+By default only the website and not the documentation is served by this environment.
+
+You can view the entire site, built with the latest documentation, with the following:
+
+```console
+$ export ENVOY_COMMIT="$(bazel run //docs:latest_version)"
+$ bazel run --action_env=ENVOY_COMMIT //site:live_docs
 ```
 
-2. Install Jekyll and other dependencies defined in the Gemfile:
+### Running the site locally (Docker)
 
-```shell
-bundle install --path vendor/bundle
+You can run the website inside a Docker container with the provided [compose recipe](docker-compose.yml).
+
+```console
+$ docker-compose up live
 ```
 
-3. Run your Jekyll site locally:
+By default only the website and not the documentation is served by this environment.
+
+You can view the entire site, built with the latest documentation, with the following:
 
-```shell
-bundle exec jekyll serve --livereload
+```console
+$ docker-compose up live_docs
 ```
 
-4. Preview the site in your web browser at http://localhost:4000.
+### Building the site locally (Bazel)
 
-### Running the site locally using Docker
+The Bazel target to build the entire website is:
+
+```console
+$ export ENVOY_COMMIT="$(bazel run //docs:latest_version)"
+$ bazel build --action_env=ENVOY_COMMIT //site:html
+```
 
-To run the website locally using Docker, run the command:
+There is a convenience script (as used in CI) that will build into a `_site` folder in the current
+directory:
 
-```shell
-docker run -it -v $(pwd):/srv/jekyll -p 4000:4000 jekyll/jekyll jekyll serve --watch --incremental
+```console
+$ ./build-website.sh
 ```
 
-Alternatively, use Docker Compose with:
+### Building the site locally (Docker)
 
-```shell
-docker-compose up
+The following command will build the entire website, including all documentation, into a `_site` folder in the current
+directory:
+
+```console
+$ docker-compose run build
 ```
 
-Preview the site in your web browser at http://localhost:4000.
+### Running Bazel commands inside a container.
+
+To drop into a Docker container, with the port mappings configured in the [composition](docker-compose.yml) file:
+
+```console
+$ docker-compose run -p 4000:4000 live bash
+```
 
-### Deploying to Github Pages
+From there you can run bazel commands directly, for example:
 
-To deploy your changes all you have to do is push to `master` and Github pages will automatically run `jekyll build` and
-deploy the generated files.
+```console
+
+me@73deecbfaf55:/src/workspace/envoy-website$ bazel run //docs:latest_version
+...
+b200312ddcbc4d237cd197a42cdd8c66cc8c6af0
+
+```
 
 ### Site content
 
 Item | Path
 :----|:----
 Home page content | [`_data/home.yml`](./_data/home.yml)
 Nav links on the main page | [`_data/nav.yml`](./_data/nav.yml)
+Documentation versions | [`_data/versions.yaml`](./_data/versions.yml)
```

Comment:
@mattklein123 @alyssawilk @RyanTheOptimist just flagging this change to the release process...

where we previously edited `_config.yml` now we need to edit `_data/versions.yml`